src/sql/connection/connection.py

```diff
@@ -829,9 +832,20 @@ def raw_execute(self, query, parameters=None, with_=None):
                         named_params_ = ", ".join(named_params)
                         e.add_detail(
                             f"Your query contains named parameters ({named_params_}) "
-                            "but the named parameters feature is disabled. Enable it "
-                            "with: %config SqlMagic.named_parameters=True"
+                            'but the named parameters feature is "warn". \nEnable it '
+                            'with: %config SqlMagic.named_parameters="enabled" \nor '
+                            "disable it with: "
+                            '%config SqlMagic.named_parameters="disabled"\n'
+                            "For more info, see the docs: "
+                            "https://jupysql.ploomber.io/en/latest/api/configuration.html"  # noqa
```

Comment:
add `#named-parameters` to the link so users are taken to a specific section

Reply:
[Added #named-parameters](https://github.com/ploomber/jupysql/pull/980/files#diff-62e3d6289e3242b271e6df3a7f201ab003f8ce73a5daf1a824cc2b4d892513d1R840)